### ISSUE TITLE
Only start time zone is set correctly on change in Calendar 2.0.1

### ISSUE THREAD

**Issue by johanehnberg** [nextcloud/calendar#1886]
### Steps to reproduce
1. Create a new event with datetime, length, 1 hour
2. Change the starting time zone to something ahead of your zone (e.g. in Germany, choose Japan)
3. Change the ending time zone to the same as 2.

### Expected behaviour
Both start and end time zones are set correctly.

### Actual behaviour
Only start time zone is set correctly, resulting in the event also not being one hour. The ending time zone is your home time zone.

### Calendar app
**Calendar app version:**
2.0.1

### Client configuration
**Browser:**
Firefox 72

**Operating system:**
Ubuntu 18.04 LTS

### Server configuration

**Operating system**:
Ubuntu 18.04 LTS

**Web server:**
Stock Apache

**Database:**
MySQL

**PHP version:**
7.2.24

**Nextcloud Version:**
17.0.2

**Updated from an older installed version or fresh install:**
Updated since ~10

**Comment by georgehrke** [comment]
@johanehnberg Can you please provide browser / javascript logs? Thx

**Comment by johanehnberg** [comment]
Sure, attached.
[console-export-2020-1-22_14-57-46.txt](https://github.com/nextcloud/calendar/files/4097381/console-export-2020-1-22_14-57-46.txt)

**Comment by georgehrke** [comment]
```
too much recursion InternalError: "too much recursion"
    t lockable.js:35
    t observer.js:34
    t abstractValue.js:41
    t calendar.js:2
    value dateTimeValue.js:414
    value dateTimeValue.js:404
    get eventComponent.js:86
    value eventComponent.js:51
    get eventComponent.js:90
    value eventComponent.js:51
    get eventComponent.js:90
main.esm.js:5221:16
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 16 globals.js:26:15
NONE RepeatFreqInterval.vue:63
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 22 globals.js:26:15
NONE RepeatFreqInterval.vue:63
The escapeHTML library is deprecated! It will be removed in nextcloud 19. 4 globals.js:26:15
too much recursion InternalError: "too much recursion"
    t lockable.js:35
    t observer.js:34
    t abstractValue.js:41
    t calendar.js:2
    value dateTimeValue.js:414
    value dateTimeValue.js:404
    get eventComponent.js:86
    value eventComponent.js:51
    get eventComponent.js:90
    value eventComponent.js:51
    get eventComponent.js:90
main.esm.js:5221:16
```

Seems relevant here.

**Comment by georgehrke** [comment]
I'm no longer able to reproduce this.

This must have been fixed along [another bug](https://github.com/nextcloud/calendar/issues/1899) (with an identical error message): 
https://github.com/georgehrke/calendar-js/commit/19969d57171df835bb091e4217b629a8b17fb410

**Comment by johanehnberg** [comment]
It still happens on 18.0.1 with 2.0.1

**Comment by georgehrke** [comment]
Which is not much of a surprise, since we released 2.0.1 on January 20th and https://github.com/georgehrke/calendar-js/commit/19969d57171df835bb091e4217b629a8b17fb410 was committed on January 24th :) 

=> Fix will come with 2.0.2
